CONTRIBUTING.md

```diff
@@ -27,6 +27,8 @@ When submitting a bug report, please try to provide as much detail as possible.
 - relevant outputs
 - any other information you deem relevant
 
+When creating or contributing to an issue, please include a link to any relevant discussion threads on [Zulip](https://chat.fhir.org) (see below).
```

Comment:
GitHub won't let me comment on line 18 above, but perhaps we should add something like:
> If you need help with using FHIR Shorthand or its tools, however, you may want to consider posting on Zulip first (see below). Questions posted on Zulip will reach a broader set of FSH users and will likely receive more timely responses. Requests for specific changes and/or bug fixes, however, should be submitted as GitHub issues so they can be formally tracked.

Reply:
I like that. I'll add it.

I was also thinking about if there's a way to combine these two separate "Issues" vs "Zulip" sections. I hadn't come up with one, but if you had also thought about it, maybe it is worth it.

Reply:
While there is considerable overlap, I think the functionality and purpose of each is distinct enough to warrant their own sections.

Reply:
I added your suggestion in [301ccea](https://github.com/FHIR/sushi/pull/1474/commits/301ccea676976b54a1539cc94dfec04360e0a84b). The only change I made was that I changed "Requests for specific changes and/or bug fixes" to just "Requests for specific changes" and let "changes" encompass all the things we list in the first sentence.